l10nzh.properties

```diff
@@ -2,16 +2,22 @@ category=Localization
 description=SonarQube Chinese Pack
 homepageUrl=https://github.com/SonarCommunity/sonar-l10n-zh
 archivedVersions=
-publicVersions=9.9,10.0,10.1,10.2,10.3,10.4
+publicVersions=9.9,10.0,10.1,10.2,10.3,10.4,10.5
 
 defaults.mavenGroupId=org.codehaus.sonar-plugins.l10n
 defaults.mavenArtifactId=sonar-l10n-zh-plugin
 
+10.5.description=Support SonarQube 10.5
+10.5.sqVersions=[10.5,LATEST]
+10.5.date=2024-04-13
+10.5.changelogUrl=https://github.com/xuhuisheng/sonar-l10n-zh/releases/tag/sonar-l10n-zh-plugin-10.5
+10.5.downloadUrl=https://github.com/xuhuisheng/sonar-l10n-zh/releases/download/sonar-l10n-zh-plugin-10.5/sonar-l10n-zh-plugin-10.5.jar
+
 10.4.description=Support SonarQube 10.4
-10.4.sqVersions=[10.4,LATEST]
-10.4.date=2024-02-10
-10.4.changelogUrl=https://github.com/xuhuisheng/sonar-l10n-zh/releases/tag/sonar-l10n-zh-plugin-10.4
-10.4.downloadUrl=https://github.com/xuhuisheng/sonar-l10n-zh/releases/download/sonar-l10n-zh-plugin-10.4/sonar-l10n-zh-plugin-10.4.jar
+10.4.sqVersions=[10.4,10.4.*]
+10.4.date=2024-03-12
+10.4.changelogUrl=https://github.com/xuhuisheng/sonar-l10n-zh/releases/tag/sonar-l10n-zh-plugin-10.4.1
```

Comment:
Okay, I see now that you've released a new binary for 10.4. In all the other changes, I overlooked that yesterday.

Please 
* _Fully revert_ 10.4 to what it was 
* Except the compatibility range. That's currently what it needs to be.
* Put it in the `archivedVersions` list.
* And add a new version for your 10.4.1 binary that's compatible with `[10.4,10.4.*]` (just like 10.4).